src/DefaultBuilder/src/WebApplicationBuilder.cs

```diff
@@ -59,6 +61,14 @@ internal WebApplicationBuilder(WebApplicationOptions options, Action<IHostBuilde
 
         bootstrapHostBuilder.ConfigureWebHostDefaults(webHostBuilder =>
         {
+            webHostBuilder.ConfigureServices(services =>
+            {
+                services.Configure<RouteOptions>(routeOptions =>
+                {
+                    routeOptions.SetParameterPolicy<RegexInlineRouteConstraint>("regex");
+                });
+            });
+
```

Comment:
```suggestion
```

This is no longer necessary now that ConfigureWebHostDefaults is doing this for us by calling WebHost.ConfigureWebDefaults before calling this. ConfigureWebHostDefaults is public API, so if it didn't add back the regex constraint back, it'd be a regression.

https://github.com/dotnet/aspnetcore/blob/2e412a8f8c46c7b93428499b5123cfa851e44f1a/src/DefaultBuilder/src/GenericHostBuilderExtensions.cs#L58-L66